data/extensions/Spells/Commands/InfoCommand.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Linq;
+using NeoServer.Game.Combat.Spells;
+using NeoServer.Game.Common;
+using NeoServer.Game.Common.Contracts.Creatures;
+using NeoServer.Game.Common.Contracts.Items;
+using NeoServer.Game.Common.Contracts.Items.Types;
+using NeoServer.Game.Common.Creatures;
+using NeoServer.Game.Items;
+using NeoServer.Networking.Packets.Outgoing;
+using NeoServer.Server.Common.Contracts;
+using NeoServer.Server.Helpers;
+
+namespace NeoServer.Extensions.Spells.Commands;
+
+public class InfoCommand : CommandSpell
+{
+    public override bool OnCast(ICombatActor actor, string words, out InvalidOperation error)
+    {
+       error = InvalidOperation.NotPossible;
+        
+       if (Params.Length != 1)
+            return false;
+
+       var ctx = IoC.GetInstance<IGameCreatureManager>();
+       ctx.TryGetPlayer(Params[0].ToString(), out var target);
+
+       if (target is null)
+           return false;
+       
+       var info = CreateItemInfo();
+       
+       var message =
+           $"""
+            AccountId: {target.AccountId}
+            Position: {target.Location.X}, {target.Location.Y}, {target.Location.Z}
+            Capacity: {target.TotalCapacity}
+            PremiumTime: {target.PremiumTime}
+            Level: {target.Level}
+            Skills:
+            {target.Skills.Where(item => item.Key != SkillType.Level).Select(Item =>   "   * " + Item.Key + ": " + Item.Value.Level + "\n").Aggregate((a, b) => a + b)}
+            """;
+
+       var window = new ListCommandsCommand.TextWindow(info, target.Location, message);
+       
+       window.WrittenBy = "OpenCoreMMO - SERVER";
+       window.WrittenOn = DateTime.Now;
+
+       var player = actor as IPlayer;
+       
+       player.Read(window);
+
+       return true;
+    }
+    
+    private static IItemType CreateItemInfo()
+    {
+        var item = new ItemType();
+        item.SetName("Info Status");
+        item.SetClientId(2821);
```

Comment:
I'm not sure what that means, but if we create a Enum to handle that ?

Reply:
This approach is similar to the Command "ListCommandSpell" created for you.  I tried to understand this on code The Baiak, but I didn't have success.